packages/web-react/src/components/TextField/README.md

```diff
@@ -42,6 +42,52 @@ TextField with password toggle (button to reveal the password):
 />
 ```
 
+## Input Width
+
+There are several ways to adjust the input width:
+
+### `inputWidth` Attribute
+
+The `inputWidth` attribute is supported on inputs of the following types: `email`,
+`password`, `tel`, `text`, `url`.
+
+```jsx
+<TextField
+  id="textFieldWidthCh"
+  inputWidth={4}
+  label="Input width wide 4 characters"
+  name="textFieldWidthCh"
+  placeholder="Placeholder"
+  type="text"
+/>
+```
+
+This option is generally recommended for inputs with a limited value length
+(e.g. numeric representation of day, month, year). Supported values are `2`, `3`
+and `4` (characters).
```

Comment:
Can you please be more specific? I do not understand why only these values are supported. In the case of day, month, and year it is applicable but in all other cases, it could be any number. So the modification of the default value makes no sense to me. Could you, please, provide some source to enlighte me? :-)

Reply:
The reason is it must be hard-coded in the CSS. I agree it can be noted here.